pysph/sph/boundary/base.py

```diff
@@ -0,0 +1,318 @@
+"""
```

Comment:
The "boundary" package could be "inlet_outlet" instead?  Boundary is too general?  Or do you intend to add more things into boundary, perhaps instead of "boundary" then we could call it "bc"?

Reply:
In any case, this file should be called `inlet_outlet_manager.py` or `inlet_outlet.py`.

Reply:
done